### PR TITLE
first pass at nwo generator

### PR DESCRIPTION
**Pull request by jctanner**

### CODE REVIEW

File: scenarios/nwo/arista.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+eos:
+  modules:
```
Comment (bcoca) [inline scenarios/nwo/arista.yml]: FYI I'm moving these from bcs to symlinks in common/, see https://github.com/ansible-community/collection_migration/tree/updates, mostly to allow reuse for those that want to capitalize on my updates

File: generate_nwo.py [hunk 2]
```diff
@@ -0,0 +1,1035 @@
...
+    'https://github.com/ansible-network/ansible_collections.junipernetworks.junos',
+    'https://github.com/ansible-network/ansible_collections.arista.eos',
+    'https://github.com/ansible-network/ansible_collections.cisco.ios',
+    'https://github.com/ansible-network/ansible_collections.vyos.vyos',
+    'https://github.com/ansible-network/ansible_collections.network.netconf',
+    'https://github.com/ansible-network/ansible_collections.network.cli',
+    'https://github.com/ansible-network/ansible_collections.cisco.nxos',
+    # community
+    'https://github.com/ansible-collections/ansible_collections_netapp',
+    'https://github.com/ansible-collections/grafana',
```
Comment (gundalow) [inline generate_nwo.py]: FYI kubernetes repo has been created, though doesn't have content in yet.

File: scenarios/nwo/ansible.yml [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+  - packaging/os/apt_repo.py
+  - packaging/os/apt_repository.py
+  - packaging/os/dnf.py
+  - packaging/os/dpkg_selections.py
+  - packaging/os/package.py
+  - packaging/os/package_facts.py
+  - packaging/os/rpm_key.py
+  - packaging/os/yum.py
+  - packaging/os/yum_repository.py
+  - remote_management/ipmi/ipmi_boot.py
```
Comment (gundalow) [inline scenarios/nwo/ansible.yml]: I don't remember seeing these two before, I'm not sure which use-case they fall under.

Or how you say these two are inscope, though other remote management standards aren't.

File: scenarios/nwo/ansible.yml [hunk 4]
```diff
@@ -0,0 +1,370 @@
...
+  - facts/virtual/netbsd.py
+  - facts/virtual/openbsd.py
+  - facts/virtual/sunos.py
+  - facts/virtual/sysctl.py
+  - json_utils.py
+  - k8s/raw.py
+  - net_tools/nios/api.py
+  - network/common/config.py
+  - network/common/netconf.py
+  - network/f5/urls.py
```
Comment (gundalow) [inline scenarios/nwo/ansible.yml]: ?

File: scenarios/nwo/ansible.yml [hunk 5]
```diff
@@ -0,0 +1,370 @@
...
+  - utilities/logic/pause.py
+  - utilities/logic/set_fact.py
+  - utilities/logic/set_stats.py
+  - utilities/logic/wait_for.py
+  - utilities/logic/wait_for_connection.py
+  module_utils:
+  - _text.py
+  - ansible_release.py
+  - api.py
+  - aws/core.py
```
Comment (gundalow) [inline scenarios/nwo/ansible.yml]: Not sure about these two AWS modules.
Do none AWS things depend on them? If so they should be rewritten.

File: scenarios/nwo/ansible.yml [hunk 6]
```diff
@@ -0,0 +1,370 @@
...
+  - facts/virtual/sunos.py
+  - facts/virtual/sysctl.py
+  - json_utils.py
+  - k8s/raw.py
+  - net_tools/nios/api.py
+  - network/common/config.py
+  - network/common/netconf.py
+  - network/f5/urls.py
+  - network/netconf/netconf.py
+  - network/skydive/api.py
```
Comment (gundalow) [inline scenarios/nwo/ansible.yml]: ?

File: scenarios/nwo/ansible.yml [hunk 7]
```diff
@@ -0,0 +1,370 @@
...
+  - common/validation.py
+  - common/warnings.py
+  - compat/paramiko.py
+  - connection.py
+  - csharp/Ansible.AccessToken.cs
+  - csharp/Ansible.Basic.cs
+  - csharp/Ansible.Become.cs
+  - csharp/Ansible.Privilege.cs
+  - csharp/Ansible.Process.cs
+  - ecs/api.py
```
Comment (gundalow) [inline scenarios/nwo/ansible.yml]: ?

File: scenarios/nwo/ansible.yml [hunk 8]
```diff
@@ -0,0 +1,370 @@
...
+  - powershell/Ansible.ModuleUtils.WebRequest.psm1
+  - service.py
+  - splitter.py
+  - urls.py
+  - yumdnf.py
+  connection:
+  - httpapi.py
+  - local.py
+  - netconf.py
+  - network_cli.py
```
Comment (gundalow) [inline scenarios/nwo/ansible.yml]: I thought this was going in netcommon?

File: scenarios/nwo/ansible.yml [hunk 9]
```diff
@@ -0,0 +1,370 @@
...
+  - powershell/Ansible.ModuleUtils.LinkUtil.psm1
+  - powershell/Ansible.ModuleUtils.PrivilegeUtil.psm1
+  - powershell/Ansible.ModuleUtils.SID.psm1
+  - powershell/Ansible.ModuleUtils.WebRequest.psm1
+  - service.py
+  - splitter.py
+  - urls.py
+  - yumdnf.py
+  connection:
+  - httpapi.py
```
Comment (gundalow) [inline scenarios/nwo/ansible.yml]: I thought this was going in netcommon?

File: scenarios/nwo/sensu.yml [hunk 10]
```diff
@@ -0,0 +1,10 @@
+sensu_go:
```
Comment (gundalow) [inline scenarios/nwo/sensu.yml]: This doesn't look right.

File: scenarios/nwo/ansible.yml (hunk 3 above)
Comment (bcoca) [reply scenarios/nwo/ansible.yml]: I used these to initialize my test server, why i kept them always in core, probably were just copied from my list

**Pull request merged**